CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # TimescaleDB Changelog
 
+## 0.1.0 (2017-06-28)
+
+**IMPORTANT NOTE**
+Starting with this release, TimescaleDB will now
+support upgrading between extension versions using the typical
+`ALTER EXTENSION` command, unless otherwise noted in future release notes. This
+important step should make it easier to test TimescaleDB and be able
+to get the latest benefits from new versions of TimescaleDB. If you
+were previously using a version with the `-beta` tag, you will need
+to `DROP` any databases currently using TimescaleDB and re-create them
+in order to upgrade to this new version. To backup and migrate data,
+use `pg_dump` to save the table schemas and `COPY` to write hypertable
```

Comment:
Should we link to http://docs.timescale.com/getting-started/setup/migrate-from-postgresql#different-db